### PR TITLE
feat: add images for core components

### PR DESCRIPTION
**Pull request by giusepperotella**
Added images for the following components:
- **coredns**: v1.6.7, v1.7.0
- **kube-apiserver**: v1.18.14, v1.20.15
- **kube-controller-manager**: v1.18.14, v1.20.15
- **kube-proxy**: v1.18.14, v1.20.15
- **kube-scheduler**: v1.18.14, v1.20.15

### CODE REVIEW

File: modules/on-premises/images.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+images:
+  - name: Coredns [Fury Kubernetes On-Premises]
+    source: k8s.gcr.io/coredns/coredns
+    tag:
+      - "v1.6.7"
+      - "v1.7.0"
+    destinations:
+      - registry.sighup.io/fury/on-premises/coredns
```
Comment (ralgozino) [inline modules/on-premises/images.yml:8]: To follow the other images nomenclature I believe this one should be:
```suggestion
      - registry.sighup.io/fury/on-premises/coredns/coredns
```

**Review by ralgozino — CHANGES_REQUESTED** [review]

**Review by ralgozino — APPROVED** [review]
lgtm

**Review by nutellinoit — CHANGES_REQUESTED** [review]
We need also all the images till 1.25

**Review by ralgozino — APPROVED** [review]
lgtm

**Pull request merged**